apps/sms/test/marionette/dialer_interaction_test.js

```diff
@@ -0,0 +1,40 @@
+/* global require, marionette, setup, suite, test, skip, __dirname */
+'use strict';
+
+var assert = require('chai').assert;
+var Messages = require('./lib/messages.js');
+var ThreadList = require('./lib/views/thread_list');
+var ThreadMock = require('./mocks/mock_message_thread');
+
+marionette('Dialer', function() {
+  var messagesApp;
+
+  var client = marionette.client({
+    prefs: {
+      'focusmanager.testmode': true
+    }
+  });
+
+  var thread;
+
+  setup(function() {
+    messagesApp = Messages.create(client);
+    ThreadMock.injectDependencies(client);
+
+    var numberOfMessages = 1;
+    thread = ThreadMock.generateThread(numberOfMessages);
+
+    messagesApp.launch();
+    messagesApp.setStorage([thread], numberOfMessages);
+
+  })
+
+  suite('Call button', function() {
+    test('should go to dialer with the correct phone number', function() {
+      var threadList = new ThreadList(messagesApp);
```

Comment:
As this is linked to the initialization of subpages, let's move this discussion on general thread.